### PR TITLE
Ome zarr and open organelle fixes

### PR DESCRIPTION
**Pull request by KateMoreva**
- OpenOrganelle 
  -  correct name loafing 

- OmeZarr
  - extra log reduced 
  - pixel size fix

### CODE REVIEW

File: src/main/java/de/embl/cba/mobie/n5/zarr/CustomZarrXmlIoSpimData.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package de.embl.cba.mobie.n5.zarr;
```
Comment (constantinpape) [inline src/main/java/de/embl/cba/mobie/n5/zarr/CustomZarrXmlIoSpimData.java:1]: What is the purpose of this class?

File: src/main/java/de/embl/cba/mobie/n5/zarr/XmlN5OmeZarrImageLoader.java [hunk 2]
```diff
@@ -27,6 +39,40 @@ public N5OMEZarrImageLoader fromXml(Element elem, File basePath, AbstractSequenc
         return null;
     }
 
+//    private SpimData openBdvZarrData(String path) {
```
Comment (constantinpape) [inline src/main/java/de/embl/cba/mobie/n5/zarr/XmlN5OmeZarrImageLoader.java:42]: Why is this function commented out?

**Review by constantinpape — COMMENTED** [review]
The changes to fix name loading, reducing the log and loading the pixel size look good.
But I am not sure why we need `CustomZarrXmlIoSpimData` and why there is a commented out function added.

File: src/main/java/de/embl/cba/mobie/n5/zarr/XmlN5OmeZarrImageLoader.java (hunk 2 above)
Comment (KateMoreva) [reply src/main/java/de/embl/cba/mobie/n5/zarr/XmlN5OmeZarrImageLoader.java:42]: it is WIP refactoring to make MoBIE code easier to read.

**Comment by tischi** [comment]
Can we merge this?

**Comment by KateMoreva** [comment]
> Can we merge this?

It should not break anything, thus yes.

**Pull request merged**